### PR TITLE
Adds edit button

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds a new edit button on the map cards:

pickme

Original ticket: #3942

### CODE REVIEW

**Comment by javierarce** [comment]
CC: @viddo + @saleiva

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4187/))

**Comment by xavijam** [comment]
Will we add this feature to public maps when user is logged?

**Comment by saleiva** [comment]
Where is that cool transition I was hoping from u? :P
welcome back dude

**Comment by javierarce** [comment]
Hi! :) 

Ha, I thought about adding a little animation, but then I realised that it could be a bit distracting.

**Comment by saleiva** [comment]
Put your sign in that job!

**Comment by javierarce** [comment]
Ok :)

**Comment by javierarce** [comment]
Ok :)

File: app/assets/stylesheets/common/map-card.css.scss [hunk 1]
```diff
@@ -220,6 +220,31 @@ $opacityTransition: 250ms;
...
 .MapCard-itemTitlePermission { background-color: $cStructure-mainBkg }
+.MapCard-editButton {
+  $size: 50px;
+  position:absolute;
+  top: -$size/2; 
+  right: 18px;
+  text-align: center;
+  width: $size;
+  height: $size;
+  background: $cCard-editButton;
```
Comment (viddo) [inline app/assets/stylesheets/common/map-card.css.scss]: If this color is only used here it's probably better to just inline it, the purpose of using the variable would be to define some conceptual color that is used in other places.

**Comment by viddo** [comment]
:+1: + not necessary to change now but for next time use a space between `property: value`, makes it more consistent with how the CSS rules are usually written.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4203/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4215/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4216/))

**Pull request merged**